components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -129,29 +130,73 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+#pragma mark - UIViewController
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  self.contentView.frame = self.view.bounds;
+  self.contentViewLabel.center = self.contentView.center;
+  CGRect exampleListTableViewFrame =
+      CGRectMake(0, self.view.bounds.size.height - exampleListTableViewHeight,
+                 self.view.bounds.size.width, exampleListTableViewHeight);
+  self.exampleListTableView.frame = exampleListTableViewFrame;
+
+  [self.bannerView sizeToFit];
+
+  // Adjust bannerViewContainer's frame
+  CGFloat topAreaInset = 0.0f;
+  if (@available(iOS 11.0, *)) {
+    topAreaInset = self.view.safeAreaInsets.top;
+  }
+
+  self.bannerView.frame = CGRectMake(0.0f, topAreaInset, self.bannerView.frame.size.width,
```

Comment:
Please use `CGRectGetWidth()` and `CGRectGetHeight()` methods throughout. Be sure not to directly access the `size` or `origin` values of a `bounds` or `frame` rect without standardizing.

Reply:
Thanks for pointing. I missed this one. Addressed in 412aafcaf84a309859e76cb3e029c771f64fd206.